CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-19.8.6
+
+* CLI
+  * Fixed missing `clusterDomain` in render RenderTapOutputProfile
+  * Added LINKERD2_PROXY_DESTINATION_GET_SUFFIXES env variable (thanks @arminbuerkle!)
```

Comment:
similar to @scottcarol's comment around line 6, combine lines 4 and 5 as they're from the same PR, and summarize into a more user-facing comment, something like:
```
Fixed custom cluster domain support for tap profiles
```